src/Tizen.NUI/src/public/XamlBinding/ContentPage.cs

```diff
@@ -158,6 +159,7 @@ protected override void Dispose(DisposeTypes type)
                 Root.Dispose();
                 Root = null;
             }
+            Window.Instance.RemovePage(this);
```

Comment:
This code makes this class unusable before Window.Instance is initialized 
it cannot be used in static constructor or initialize of static member.
Is this by design?